app/services/iiif_print/pluggable_derivative_service.rb

```diff
@@ -106,6 +106,7 @@ def impending_derivative?(name)
   # "valid?" which would now look at the specific file_set for validity.
   def plugins_for(file_set)
     parent = parent_for(file_set)
+    return Array(default_plugins) unless file_set.mime_type.match?(/image/)
```

Comment:
Do we want to repurpose 

https://github.com/samvera/hyrax/blob/76176286b817c869a80b922ccacc6fecea3827e2/app/services/hyrax/file_set_derivatives_service.rb#L38

Reply:
oh perfect, didn't know about that one!

Reply:
Also, I think our existing `valid?` methods on the various classes should be doing this test.